coreutils/Makefile

```diff
@@ -410,3 +410,48 @@ exp-201609:
  		make $$program.tracerx3; \
 		make $$program.tracerx4; \
 	done
+
+# This target is used for collecting data from experiment exp-201609 result into log-exp-201609.csv file.
+csv-exp-201609:
+	rm -f log-exp-201609.csv
+	echo "Directory,Time, #instructions,#completed paths,#error paths,#subsumed paths,Average branching depth of completed paths,Average branching depth of subsumption paths,Average instructions of completed paths, Average instructions of subsumed paths,Icov,Bcov,Gcov,SLOC" >> log-exp-201609.csv; \
+	for program in cut expand fmt fold join pr printf sort sum tail tsort uniq wc factor seq; \
```

Comment:
Here the loop that iterates over `ext` should be the outer one, and the loop that iterates over `program` the inner loop. The experimental table will be easier to read with if the runs are first categorized according to the tool first before the individual problem.